packages/client/src/views/Keywords.vue

```diff
@@ -35,12 +50,31 @@ export default {
       fields: [
         {
           key: 'search_term',
+          thStyle: { width: '20%' },
         },
         {
           key: 'notes',
+          thStyle: { width: '50%' },
+        },
+        {
+          key: 'created_at',
+          thStyle: { width: '20%' },
+          formatter: (value) => {
+            const date = new Date(value);
+            const month = date.getMonth() + 1;
+            const day = date.getDate();
+            const year = date.getFullYear();
+
+            return `${month}/${day}/${year}`;
```

Comment:
Could be simplified to-
```js
return moment(value, 'YYYY-MM-DD').format('M/DD/YYYY');
```

In line 43 you'll need to add `import moment from 'moment';`

Reply:
Good call!